src/api_server/src/request/machine_configuration.rs

```diff
@@ -93,16 +120,34 @@ mod tests {
         }
 
         let body = r#"{
-                "vcpu_count": 8,
-                "mem_size_mib": 1024,
-                "smt": false,
-                "track_dirty_pages": true
-            }"#;
+            "vcpu_count": 8,
+            "mem_size_mib": 1024,
+            "cpu_template": "None"
+        }"#;
         let expected_config = MachineConfigUpdate {
             vcpu_count: Some(8),
             mem_size_mib: Some(1024),
             smt: Some(false),
             cpu_template: Some(StaticCpuTemplate::None),
+            track_dirty_pages: Some(false),
+        };
+
+        match vmm_action_from_request(parse_put_machine_config(&Body::new(body)).unwrap()) {
+            VmmAction::UpdateVmConfiguration(config) => assert_eq!(config, expected_config),
+            _ => panic!("Test failed."),
+        }
```

Comment:
nit:
```suggestion
        assert_eq!(vmm_action_from_request(parse_put_machine_config(&Body::new(body)).unwrap()), VmmAction::UpdateVmConfiguration(expected_config));
```

Reply:
This pattern is being used in many places, so I'll apply this in another commit. Thanks!

Reply:
done.

Reply:
Aaaa, sorry, didnt meant to make you go through the entire codebase. But Thanks!!